test/specs/simulation.random.spec.js

```diff
@@ -75,3 +76,34 @@ describe('randomMissionsCompleted()', () => {
     ).toBeGreaterThanOrEqual(4);
   });
 });
+
+describe('randomCoords()', () => {
+  const sampleArguments = { coords: { lat: 1, long: 1 }, radius: 1000 };
+  const random = randomCoords(sampleArguments);
+  const coords = {
+    lat1: sampleArguments.coords.lat,
+    long1: sampleArguments.coords.lat,
+    lat2: random.lat,
+    long2: random.long
+  };
+  /**
+   * Haversine Formula: returns the distance from one point to another
+   * @returns {Number} a number containing distance in meters
+   */
+  const getDistance = ({ lat1, long1, lat2, long2 }) => {
```

Comment:
Look at you getting all geodesic in here 😄 👍 

One suggestion. Remove the haversine from there and use the existing dependency on turf to calculate the distance. No need to have another duplicate of that logic in the test file. You can see it in action here: https://github.com/DAVFoundation/missioncontrol/blob/da2e50fec635a30223452fbc4458e1adeba0956d/server/simulation/vehicles.js#L12

Reply:
Ahh yes, makes sense that there would be a dependency for that 😆